tests/integration/query_helpers/test_query_docs.py

```diff
@@ -0,0 +1,125 @@
+"""Integration tests using light weight FAISS index."""
```

Comment:
This is not a mock vector store. mock vector is in unit test cases. This is an index. Anyhow if you dont write integration test for your components it will fail the test-integration check due to insufficient test coverage. This is very common practice when you testing db integration where you write you test cases with lightweight db.